src/main/java/at/ac/fhcampuswien/fhmdb/state/DESCState.java

```diff
@@ -0,0 +1,18 @@
+package at.ac.fhcampuswien.fhmdb.state;
+
+import at.ac.fhcampuswien.fhmdb.controller.HomeController;
+
+import java.util.Comparator;
+
+public class DESCState extends State {
+    public DESCState(HomeController homeController) {
+        super(homeController);
+    }
+
+    @Override
+    public void pressSortBtn() {
+        homeController.setState(new ASCState(homeController));
+        homeController.movies.sort(Comparator.naturalOrder());
+        homeController.sortBtn.setText(SORT_DEFAULT_TEXT_DESC);
+    }
```

Comment:
if this overridden method is exactly the same as the one in `DefaultState` then you don't need to override it

Reply:
How would I not need to override it? The method is derived from State not from DefaultState.

Reply:
sorry, my comment was incomplete and confusing. I meant to say that we can have the default behaviour in the State.pressSortBtn() method and only override it in the DESCState. Basically we would clean some duplicated code from ASCState and DefaultState